### PR TITLE
netdevices.mk: add niu device

### PR DESCRIPTION
**Pull request by lucize**
Compile/Run tested: x86_64 Reboot (SNAPSHOT, r5226-986d9deb3b)

### CODE REVIEW

File: package/kernel/linux/modules/netdevices.mk [hunk 1]
```diff
@@ -559,6 +575,25 @@ endef
 $(eval $(call KernelPackage,ixgbe))
 
 
+define KernelPackage/ixgbevf
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
```
Comment (hauke) [inline package/kernel/linux/modules/netdevices.mk]: You add two packages , but the commit title only says you add one. Please extend the commit message

**Comment by lucize** [comment]
I thought about that too, I'll split them

**Comment by lucize** [comment]
I'll make another pr only with the sort because is giving me branch conflict if I do it in this one and I'm not so good at git

**Comment by hauke** [comment]
Thank you for the pull request, this was merged into master.

**Pull request closed without merging**